kratos/CMakeLists.txt

```diff
@@ -39,7 +39,6 @@ set( KRATOS_CORE_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/sources/standard_linear_solver_factory.cpp;
     ${CMAKE_CURRENT_SOURCE_DIR}/sources/standard_preconditioner_factory.cpp;
     #${CMAKE_CURRENT_SOURCE_DIR}/sources/kratos_exception.cpp;
-    ${CMAKE_CURRENT_SOURCE_DIR}/containers/flags.cpp;
```

Comment:
So the flags.cpp is removed from cmake but not from the folder?

Reply:
This file is not changed anymore, I think you are comparing and outdated version :S